airbyte-integrations/connectors/source-zendesk-chat/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM airbyte/integration-base-python:0.1.4
```

Comment:
update to latest 
```suggestion
FROM airbyte/integration-base-python:0.1.6
```

Reply:
Updated it in PR for Support Incremental #3157